figo/models.py

```diff
@@ -185,9 +369,9 @@ def get_security(self, security_id):
         """
         return self.session.get_security(self.account_id, security_id)
 
-    def __unicode__(self):
-        return u"Account: %s (%s at %s)" % (
-            self.name, self.account_number, self.bank_name
+    def __str__(self):
+        return (
+            f"Account: {self.name} ({self.account_number} at {self.bank_name})"
```

Comment:
I'd use a single quotes:
`f'Account: {self.name} ({self.account_number} at {self.bank_name})'`

here and in other places within this file.

Reply:
In this project from beginning dominated was double-quotes, so I will leave them as they are.